app/client/src/routes/home.tsx

```diff
@@ -728,6 +731,16 @@ function SelectFilter<
     <Select
       aria-label={`${filterLabel} input`}
       className="width-full"
+      formatOptionLabel={(option) =>
+        secondaryFilterKey ? (
+          <div>
+            {option.value}{' '}
+            <span style={{ fontWeight: 600 }}>({option.label})</span>
+          </div>
+        ) : (
+          option.label
+        )
+      }
```

Comment:
Can you fix the codesmell here? Basically, this just needs to be put into a separate function component.

Reply:
Yeah, the first way I tried didn't fix the smell, I'll try another way.

Reply:
I used a callback to fix this. I know SonarCloud advises against using a callback, but the prop in question is actually looking for a callback in this case.